BikeData/src/main/scala/by/itechart/ParsingLogic.scala

```diff
@@ -0,0 +1,93 @@
+package by.itechart
+
+import java.io.{BufferedWriter, FileWriter}
+import java.time.Month
+import java.util.concurrent.TimeUnit
+
+import au.com.bytecode.opencsv.CSVWriter
+
+import scala.collection.JavaConverters._
+import scala.io.Source
+
+object ParsingLogic {
+  type BikeInfo = String
+
+  private val table = tableReader()
+
+  //task_4
+  def generalStats() {
+    val strPath = "/general-stats.csv"
```

Comment:
Move name of generated file into config, so I could change it without changing the code. Here and in other similar places